jerry-core/ecma/base/ecma-gc.c

```diff
@@ -1899,7 +1899,9 @@ ecma_gc_free_object (ecma_object_t *object_p) /**< object to free */
 #endif /* JERRY_ESNEXT */
 #if JERRY_BUILTIN_TYPEDARRAY
         case ECMA_OBJECT_CLASS_ARRAY_BUFFER:
+        #if JERRY_BUILTIN_SHAREDARRAYBUFFER
```

Comment:
This #if-#endif should be aligned to line start (other nested #if-s are also aligned on line start)